### ISSUE TITLE
["Error:Unable to retrieve authentication token."]

### ISSUE THREAD

**Issue by miok-jung** [microsoft/AzureStorageExplorer#3645]
**Storage Explorer Version**: 
**Build Number**: 
**Platform/OS**: 
**Architecture**: 
**Regression From**:

**Comment by Psuban** [comment]
I'm experiencing the same error with my work azure account

**Comment by MRayermannMSFT** [comment]
can y'all go to Settings -> Services -> Disks, uncheck the checkbox and then restart? That should unblock you (assuming you don't need to work with disks).

**Comment by TechWatching** [comment]
This workaround worked for me. But if it wasn't this GitHub issue, I would have never solved my problem. And it's a shame it does not work in Azure portal.

**Comment by MRayermannMSFT** [comment]
Closing due to lack of response by original issue opener. For anyone still having this problem, please try the advised steps, and then if you are still blocked, please open a new issue. Thanks.
